neuraxle/base.py

```diff
@@ -93,6 +93,9 @@ def __iter__(self):
 
         return zip(current_ids, self.data_inputs, expected_outputs)
 
+    def __str__(self):
+        return ';'.join(self.current_ids)
```

Comment:
Perhaps something more like this instead? It would feel more Pythonic. 
`return self.__class__.__name__ + "(current_ids=" + repr(list(self.current_ids)) + ", ...)"`